toolchain/parse/testdata/operators/fail_precedence_as.carbon

```diff
@@ -16,16 +16,40 @@ fn F(n: i32) {
   // CHECK:STDERR:
   not true as bool;
 
-  // No ordering between most binary operators and `as`.
+  // No ordering between mathematical binary operators and `as`.
   // CHECK:STDERR: fail_precedence_as.carbon:[[@LINE+4]]:9: ERROR: Parentheses are required to disambiguate operator precedence.
   // CHECK:STDERR:   1 + 1 as i32;
   // CHECK:STDERR:         ^~
   // CHECK:STDERR:
   1 + 1 as i32;
-  // CHECK:STDERR: fail_precedence_as.carbon:[[@LINE+3]]:9: ERROR: Parentheses are required to disambiguate operator precedence.
+  // CHECK:STDERR: fail_precedence_as.carbon:[[@LINE+4]]:9: ERROR: Parentheses are required to disambiguate operator precedence.
   // CHECK:STDERR:   5 % 2 as i32;
   // CHECK:STDERR:         ^~
+  // CHECK:STDERR:
```

Comment:
I don't understand how the code changes in this PR could add a line of whitespace to this diagnostic. This probably shouldn't block the PR, but it's very odd.

Reply:
We place blank lines between diagnostics. Since a later diagnostic is added to the file, and a blank line has no line number association, the blank line is associated with the prior diagnostic.